arviz/plots/jointplot.py

```diff
@@ -29,8 +29,9 @@ def plot_joint(
     data : obj
         Any object that can be converted to an az.InferenceData object
         Refer to documentation of az.convert_to_dataset for details
-    var_names : Iter of 2 e.g. (var_1, var_2)
-        Variables to be plotted, two variables are required.
+    var_names : tuple
+        Variables to be plotted.
+        Two variables(or one variable with multiple dimensions) are required.
```

Comment:
There are actually 2 cases only: either 2 variables or 1 variable (in which case coords must select a subset of exactly 2 dimensions). The type could show this with something like ` var_names : str or iterable of str` and the description explain these 2 cases.

Reply:
@OriolAbril Okay I'll do these changes.